packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -284,6 +327,40 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
           </Row>
         </>
       )}
+      {showRelOption && (
```

Comment:
Nit: If instead we'd wrap all this into `displayMetaFields` as well together with the `Row` above, `showRelOptions` could be just `MULTIPLE_VALUE !== link.rel`.

Not a big difference but might improve readability.